types/loadable__server/index.d.ts

```diff
@@ -2,6 +2,7 @@
 // Project: https://github.com/smooth-code/loadable-components
 // Definitions by: Martynas Kadiša <https://github.com/martynaskadisa>
 //                 Luis Herranz <https://github.com/luisherranz>
+//                 Arman Sargsyan <https://github.com/Armanio>
```

Comment:
Just to confirm: this list isn't for crediting contributors - it's a list of people who want to be notified when this package changes so that they can provide reviews.

Reply:
Oh! My bad. It is now clear. Thx for explain.